platform_db_backup.sh

```diff
@@ -61,23 +61,23 @@ if [[ -v AWS_BACKUP_BUCKET || -v SFTP_SERVER ]]; then
     if [[ -v SFTP_DIRECTORY && -v SITE && -v SFTP_SERVER ]]; then
       echo "Checking old backups"
       if [[ -n "$SFTP_PORT" ]]; then
-        ssh -p $SFTP_PORT ${SFTP_USERNAME}@${SFTP_SERVER} "find ~/$SFTP_DIRECTORY/drush-backups/$PROJECT_NAME -mindepth 1 -type d -mtime +${SFTP_DAYS_EXP:180} -printf '%p\n' |grep -v '\-d01' |xargs -I {} rm -r -v \"{}\""
+        ssh -p "$SFTP_PORT" "${SFTP_USERNAME}"@"${SFTP_SERVER}" "find ~/$SFTP_DIRECTORY/drush-backups/$PROJECT_NAME -mindepth 1 -type d -mtime +${SFTP_DAYS_EXP:180} -printf '%p\n' |grep -v '\-d01' |xargs -I {} rm -r -v \"{}\""
```

Comment:
SC1117: Backslash is literal in "\n". Prefer explicit escaping: "\\n". (`printf '%p\n'`)